packages/ti_misp/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.29.2"
```

```suggestion
- version: "1.30.0"
```

Considering an enhancement

---

Done in new commit.